src/Microsoft.OData.Client/Metadata/ClientTypeUtil.cs

```diff
@@ -617,23 +658,33 @@ internal static string GetClientFieldName(Type t, string serverDefinedName)
         /// <returns>Client PropertyInfo, or null if the method is not found.</returns>

```

Comment:
>or null if the method is not found [](http://example.com/codeflow?start=42&length=34)

actually, whether this returns null or throws an exception depends on undeclaredPropertyBehavior, right?